### ISSUE TITLE
Unsaved changes indicator

### ISSUE THREAD

**Issue by PeterNerlich** [digitalfabrik/integreat-cms#2477]
### Motivation

As a user editing e.g. a page, I want to know whether I have unsaved changes, or notice when an autosave is successful.
This should result in slightly higher perceived reliability of the system through transparency and predictability.

### Proposed Solution

Add a bullet to the document title whenever the form is different from the most recently saved state.

### Alternatives

- Add a banner to the top of the page once changes have been saved
  Might get annoying, especially if it changes the layout / scroll position relative to the focused element. Very verbose compared to the minor role this indicator is supposed to achieve.
- Add an indicator to the tinymce editor
  Might be relatively complex

The proposed solution is not better than these alternatives. But it requires way less effort and is a good starting point.

### User Story

As an editor, I want to know whether I have unsaved changes, or notice when an autosave is successful, so that I have a higher trust in the system.

**Comment by osmers** [comment]
We are already introducing the new auto saved message, right? Wouldn't it then make sense to simply erase that message if there are unsaved changes? To show that the current version is not auto-saved yet?

**Comment by PeterNerlich** [comment]
@osmers I'm not up to date regarding this, do you have an issue/PR id for me?

**Comment by MizukiTemma** [comment]
I haven't checked your PR for this issue, but you mean you need this for #2401 , right?

**Comment by PeterNerlich** [comment]
@MizukiTemma No, the other way around. #2476 builds on (=contains the commit for) #2401, so after #2401 is merged, I would like this to be considered to be merged.

**Comment by osmers** [comment]
@PeterNerlich yeah, #2431 is the one I meant. Does your last comment then mean, that this issue will be closed? Though we can still discuss the "notice when I have unsaved changes" bit of your issue.

**Comment by PeterNerlich** [comment]
> Does your last comment then mean, that this issue will be closed?

I created this issue mostly because I need a valid issue ID in order to add a release note to #2476 as per our `new_release_note.sh` tool. If that PR is merged, this issue will be closed automatically as it is listed as addressing this issue.
The implemented functionality relevant to this issue kind of emerged as a byproduct of the rest of the changes I did. I included it because it was like 4 lines of code and felt really good to me. Whether we decide to use these, merge them with #2431 or reject them for it, is all fine with me. I have opinions on UX, but no strong feelings for this implementation in particular.

**Comment by dkehne** [comment]
Any more need to discuss here?

**Comment by osmers** [comment]
@PeterNerlich anything that still needs to be done here or should we close this/move it to Q1 since we have too many issues in Q4?

**Comment by PeterNerlich** [comment]
@osmers I don't think this is still going to fit in this quarter, feel free to move it
